### ISSUE TITLE
[Bug] - I clicked on "Tell me a Scary Story" then "New Story" and got an error

### ISSUE THREAD

**Issue by vlakjacket** [Monika-After-Story/MonikaModDev#8116]
Hello, it's Halloween and I asked Monika to tell me a scary story, and I got an error. How do I fix this?

I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-stories.rpy", line 136, in script call
  File "game/script-stories.rpy", line 236, in script
  File "game/script-stories.rpy", line 238, in <module>
  File "game/script-stories.rpy", line 116, in get_and_unlock_random_story
IndexError: list index out of range

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1510, in script call
  File "game/event-handler.rpy", line 2530, in script call
  File "game/script-stories.rpy", line 136, in script call
  File "game/script-stories.rpy", line 236, in script
  File "C:\Users\Mervin\Desktop\Doki Doki Literature Club\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Mervin\Desktop\Doki Doki Literature Club\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script-stories.rpy", line 238, in <module>
  File "game/script-stories.rpy", line 116, in get_and_unlock_random_story
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/random.py", line 275, in choice
IndexError: list index out of range

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.4

**Comment by arbabf** [comment]
Yep. Happened to me as well.

**Comment by multimokia** [comment]
Will be fixed in the next release, this is an o31 specific issue but the error itself can be `ignore`d, if you click that button on the traceback screen.

That said, I'd advise not to ask for scary stories if you've unlocked them all (17)

**Comment by vlakjacket** [comment]
Will do! Thank you very much for answering
